doc-content/enterprise-only/admin-and-config/maven-settings-configuration-ref.adoc

```diff
@@ -4,9 +4,26 @@
 
 {PRODUCT} uses Maven `settings.xml` file to configure it's Maven execution. You must create and activate a profile in the `settings.xml` file and declare the Maven repositories used by your {PRODUCT} projects.
 
-.Procedure
-. In the `settings.xml` file, declare the repositories that your {PRODUCT} projects use. Usually, this is either the https://maven.repository.redhat.com/ga/[online {PRODUCT} Maven repository] or the 
-{PRODUCT} Maven repository that you download from the Red Hat Customer Portal and any repositories for custom artifacts that you want to use. 
-. Ensure that {CENTRAL} or {KIE_SERVER} is configured to use the `settings.xml` file, for example by specifying the `kie.maven.settings.custom=<path_to_settings.xml>` property.
-
 For information about the Maven `settings.xml` file, see the Apache Maven Project  https://maven.apache.org/settings.html[Setting Reference].
+
+.Procedure
+. In the `settings.xml` file, declare the repositories that your {PRODUCT} projects use. Usually, this is either the https://maven.repository.redhat.com/ga/[online {PRODUCT} Maven repository] or the
+{PRODUCT} Maven repository that you download from the Red Hat Customer Portal and any repositories for custom artifacts that you want to use.
+. Ensure that {CENTRAL} or {KIE_SERVER} is configured to use the `settings.xml` file , for example by specifying the `kie.maven.settings.custom=<SETTINGS_FILE_PATH>` property where `<SETTINGS_FILE_PATH>` is the path to the `settings.xml` file.
++
+ifeval::["{context}" == "install-on-jws"]
+On {JWS}, for {KIE_SERVER} add `-Dkie.maven.settings.custom=<SETTINGS_FILE_PATH>` to the  `CATALINA_OPTS` section of the `setenv.sh` (Linux) or `setenv.bat` (Windows) file. .For standalone {CENTRAL}, enter the following command:
++
+[source]
+----
+java -jar rhpam-7.4.0-business-central-standalone.jar -s application-config.yaml -Dkie.maven.settings.custom=<SETTINGS_FILE_PATH>
+----
+endif::[]
+ifeval::["{context}" == "install-on-tomcat"]
+On {TOMCAT}, for {KIE_SERVER} add `-Dkie.maven.settings.custom=<SETTINGS_FILE_PATH>` to the  `CATALINA_OPTS` section of the `setenv.sh` (Linux) or `setenv.bat` (Windows) file. .For standalone {CENTRAL}, enter the following command:
```

Comment:
The below sentence needs to be moved to new line or the full stop removed from the start:

".For standalone {CENTRAL}, enter the following command:"